modules/data.remote/R/NASA_DAAC_download.R

```diff
@@ -0,0 +1,110 @@
+#' Parallel download data from the NASA ORNL DAAC server given period, spatial bounding box, and data DOI.
+#'
+#' @param ul_lat Numeric: upper left latitude.
+#' @param lr_lat Numeric: lower right latitude.
+#' @param ul_lon Numeric: upper left longitude.
+#' @param lr_lon Numeric: lower right longitude.
+#' @param ncore Numeric: numbers of core to be used if the maximum core
+#' @param from Character: date from which the data search starts. In the form
+#'   "yyyy-mm-dd".
+#' @param to Character: date on which the data search end. In the form
+#'   "yyyy-mm-dd".
+#' @param outdir Character: path of the directory in which to save the
+#'   downloaded files.Default to the working directory. If it doesn't exist it
+#'   will be created. Ignored if \code{just_path=TRUE}
+#' @param doi Character: data DOI on the NASA DAAC server.
```

Comment:
This is probably the most important argument and would benefit from additional info so users know where to get these and how to format them. Would also be good to have a simple example below.

Reply:
Fixed.